js/projects.js

```diff
@@ -0,0 +1,175 @@
+    popupTech: ['html', 'css', 'javaScript', 'github', 'Ruby', 'Bootstrap'],
+    seeLive: '#',
+    seeSource: '#',
+  },
+  {
+    id: 3,
+    name: 'Tonic',
+    experience: ['CANOPY', 'Back End Dev', 2015],
+    image: '/images/Snapshoot-Portfolio-3.svg',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    popDescription:
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quod accusantium porro temporibus consequatur veritatis sequi ullam. Optio, id non voluptatum ipsum distinctio sunt cum, obcaecati, eaque soluta iusto cupiditate tempora?',
+    technologies: ['html', 'Ruby on rails', 'css', 'javascript'],
+    popupTech: ['html', 'css', 'javaScript', 'github', 'Ruby', 'Bootstrap'],
+    seeLive: '#',
+    seeSource: '#',
+  },
+  {
+    id: 4,
+    name: 'Uber Navigation',
+    experience: ['Uber', 'Lead Developer', 2018],
+    image: '/images/Snapshoot-Portfolio-4.svg',
+    description:
+      'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    popDescription:
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quis alias, quidem nemo sequi sint officia dignissimos adipisci iusto eos dolore cumque eius est. Iure molestias aut velit magni non autem? ',
+    technologies: ['html', 'Ruby on rails', 'css', 'javascript'],
+    popupTech: ['html', 'css', 'javaScript', 'github', 'Ruby', 'Bootstrap'],
+    seeLive: '#',
+    seeSource: '#',
+  },
+];
+
+/* Populate object to html elements when the page loads */
+for (let i = 0; i < projects.length; i += 1) {
+  let tech = '';
+  let experienceList = '';
+
+  for (let j = 0; j < projects[i].technologies.length; j += 1) {
+    tech += `<li class="technology"><a href="#">${projects[i].technologies[j]}</a></li>`;
+  }
+
+  for (let k = 0; k < projects[i].experience.length; k += 1) {
+    experienceList
+      += k === 0
+        ? `<li><a href="#" class="canopy">${projects[i].experience[k]}</a></li>`
+        : `<li ><a href="#">${projects[i].experience[k]}</a></li>`;
+    if (k !== projects[i].experience.length - 1) {
+      experienceList
+        += '<li><a href="#"><img src="/images/Counter.png" alt="dot"></a></li>';
+    }
+  }
+
+  document.querySelector('.project-container').innerHTML += `
+    <div class="project-description" id="project-1">
+    <img src=${projects[i].image} alt="${projects[i].name}" >
+    <div class="project-details">
+        <h2>${projects[i].name}</h2>
+        <ul class="experience">
+          ${experienceList}
+        </ul>
+
+        <p>${projects[i].description}</p>
+        <ul class="technology">
+          ${tech}
+        </ul>
+      
+        <div class="btn-project">
+          <button class="btn-see-project"  data-index="${i}">
+            See Project
+          </button>
+        </div>
+     </div>  
+    </div>
+  `;
+}
+
+/* Begging of display popup modal */
+
```

Comment:
Since you are using JavaScript for this milestone, you should set up both [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters in the project repository.

Kindly make sure your `linters.yml` file is the one from the [html&css + javascript](https://github.com/microverseinc/linters-config/blob/master/html-css-js) Linters Config repository.

![image](https://user-images.githubusercontent.com/38088818/167582683-57ee2c36-0b50-4a9c-9042-a74e97e9e88b.png)

- Please make sure your ESlint linter checks are passing after updating the config files.

Your checks should look like this:
![Eslintchecks](https://user-images.githubusercontent.com/38088818/168270144-5cf68371-6354-4476-90fa-94debe7cd848.png)